melodic/distribution.yaml

```diff
@@ -13320,21 +13320,6 @@ repositories:
       url: https://gitlab.com/InstitutMaupertuis/simple_rviz_plugin.git
       version: melodic
     status: maintained
-  skyway:
```

Comment:
Please don't change `melodic/distribution.yaml`. ROS Melodic is end-of-life, so this file is essentially frozen.